Sources/IO/Misc/PDBReader/index.js

```diff
@@ -0,0 +1,257 @@
+import * as macro from '../../../macro';
+import vtkPolyData from '../../../Common/DataModel/PolyData';
+import vtkDataArray from '../../../Common/Core/DataArray';
+import DynamicTypedArray from '../../../Common/Core/DynamicTypedArray';
+
+import DataAccessHelper from '../../Core/DataAccessHelper';
+
+import atomElem from '../../../../Utilities/XMLConverter/chemistry/elements.json';
```

Comment:
Yes because we need finally more than just the `elementColor` and the `radiusCovalent`.
But I agree that we definitely don' t need the `nameOrigin` and other descriptive things.